rampwf/score_types/detection.py

```diff
@@ -438,14 +504,20 @@ def precision(y_true, y_pred, matches=None, iou_threshold=0.5,
     matches : optional, output of _match_tuples
     iou_threshold : float
         Threshold to determine match
+    minipatch : list of int, optional
+        Bounds of the internal scoring patch (default is None)
 
     Returns
     -------
     precision_score : float [0 - 1]
     """
+    if minipatch is not None:
+        y_pred = [filter_minipatch_tuples(y_patch, minipatch)
```

Comment:
Can you add something like the following

```
# precision is calculated relative to the number of predicted objects, 
# so we filter for the scoring region based on the predictions  
```

to clarify the reasoning of the code

(the same for recall)